gita/__main__.py

```diff
@@ -141,6 +141,14 @@ def f_group(args: argparse.Namespace):
             utils.write_to_groups_file(groups, 'w')
         else:
             utils.write_to_groups_file({gname: sorted(args.to_group)}, 'a+')
+    elif cmd == 'remove':
+        gname = args.gname
+        if gname in groups:
+            gname_repos = set(groups[gname])
+            for repo in args.from_group:
+                gname_repos.discard(repo)
+            groups[gname] = sorted(gname_repos)
```

Comment:
If `groups[gname]` is a list, we can delete the `repo` without sorting again

Reply:
true, this was mostly a quick copypasta. patched